### ISSUE TITLE
Criar da integração contínua do Webcrawler

### ISSUE THREAD

**Issue by guilacerda** [fga-eps-mds/2018.2-Lino#137]
### Descrição
Existe a necessidade de se automatizar o deploy do webcrawler, tanto do RU quanto do Matrícula. Essa issue refere-se à criação do CI dos dois serviços.

3 estágios devem ser observados:
* Test: estágio na qual é testado o código estático e roda os testes desenvolvidos pela equipe.
* Build: refere-se ao estágio que é feito a build do serviço, via docker, e é enviado para o registry do dockerhub (atualizando a última versão)
* Deploy: nessa etapa, é atualizada em produção os serviços do crawler. 

### Tarefas
**Webcrawler-RU**
- [ ] Fazer o mirror do repositório do Github no Gitlab
- [ ] Utilizar o gitlab-ci para a integração
- [ ] Elabora o stage de Test
- [ ] Elaborar o stage de Build
- [ ] Elabora o stage de Deploy

**Webcrawler-Matrícula**
- [ ] Fazer o mirror do repositório do Github no Gitlab
- [ ] Utilizar o gitlab-ci para a integração
- [ ] Elabora o stage de Test
- [ ] Elaborar o stage de Build
- [ ] Elabora o stage de Deploy

### Critérios de Aceitação
- [ ] Todos os stages devem estar funcionando corretamente

**Comment by guilacerda** [comment]
Foi encontrado um problema no stage de teste quando adicionado um stage a mais, hoje estamos testando estaticamente com flake8 e testes unitários. O problema encontrado foi que utilizamos o mongo como banco para realizar alguns teste unitários. Há uma comunicação entre eles, e dentro do gitlab-ci estávamos encontrando alguns problemas pra conexão. Tentamos de várias maneiras (eu e o @guilhesme23), até conseguir encontrar a solução do banco que é: iniciar o serviço do banco de dados antes de começar o stage de teste.

**Comment by guilacerda** [comment]
Identifiquei que além do que proposto na issue, tem também que criar um stágio para buildar e "deployar" o cronjob do serviço do webcrawler. Tal serviço é responsável por tempo em tempo executar alguns comandos.

### Tarefas

- [ ] Criar estágio de build para o Cronjob
- [ ] Criar estágio de deploy para o Cronjob

### Critérios de Aceitação

- [ ] O cronjob está funcionando corretamente em produção

**Comment by guilacerda** [comment]
É preferível separar a issue para o pipeline do webcrawler do RU e uma nova para o do Matrícula. Para ficar mais fácil de rastrear. O que acham? @brunapinos e @guiaugusto

**Comment by guiaugusto** [comment]
Eu acho até mais certo essa separação, pra falar a verdade... Por que objetiva mais as issues, daí tem um rastro melhor como você disse!

**Comment by brunapinos** [comment]
@guiaugusto Concordo, mas é importante colocar o link dos repositórios nas issues, só para facilitar

**Comment by guilacerda** [comment]
Então fica decidido que as tarefas referentes ao webcrawler matrícula se encontram na nova issue criada #142
